src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/JobTemplateService.scala

```diff
@@ -35,6 +35,7 @@ trait JobTemplateService {
   def createJobTemplate(jobTemplate: JobTemplate)(implicit ec: ExecutionContext): Future[JobTemplate]
   def updateJobTemplate(jobTemplate: JobTemplate)(implicit ec: ExecutionContext): Future[JobTemplate]
   def deleteJobTemplate(id: Long)(implicit ec: ExecutionContext): Future[Boolean]
+  def getJobTemplateUsage(id: Long): Future[Seq[Workflow]]
```

Comment:
I would name this `getWorkflowsByJobTemplate`, similar to `getWorkflowsByProjectName`. You can then ask yourself, whether this belongs to the `WorkflowService` or the `JobTemplateService`, but for me it's ok here.